ls1intum/Artemis
# [Exam] Add working time

### Checklist

### Motivation and Context
Individual students can receive compensation for disadvantages, i.e. they have X % more time, whereby X can have different values per student (e.g. 25, 40, 50, 100). Simple attribute workingTime (defaults to the exam duration).

### Description
- [x] I've added the attribute `workingTime` to `StudentExam`
- [x] I've ensured that the default working time will be set for all student exams during generation
- [x] The working time will be displayed in the student exam detail view

### Steps for Testing
1. Log in to Artemis
2. Generate student exams
3. Go to the detail view of a student exam and check that the displayed working time equals the duration of the exam

### Screenshots
![image](https://user-images.githubusercontent.com/15168372/85838488-ef3f3e00-b799-11ea-9f27-3ad9fc6617db.png)
